gordon/plugins_loader.py

```diff
@@ -227,8 +227,9 @@ def load_plugins(config, plugin_kwargs):
 
     active_plugins = _get_activated_plugins(config, installed_plugins)
     if not active_plugins:
-        return [], [], []
+        return [], [], [], None
     plugin_namespaces = _get_plugin_config_keys(active_plugins)
     plugin_configs = _load_plugin_configs(plugin_namespaces, config)
-    return _init_plugins(
+    plugin_names, plugins, errors = _init_plugins(
         active_plugins, installed_plugins, plugin_configs, plugin_kwargs)
+    return plugin_names, plugins, errors, metrics_plugin
```

Comment:
perfecto, thank you!

Reply:
Ah, yeah, I actually noticed it didn't fail in my next PR. Will fix.

Reply:
Looks like a case of too much mocking...

Reply:
😬